storybookjs/storybook
# Core: Support recursive addon imports

Issue: #19176

## What I did

This is an attempt to implement the idea in #19176 to pre-require the sub-addons of addon-essentials, so that the preview builders don't need to, thus avoiding problems with pnpm and yarn pnp.  

However, after we do import those, we have the problem of getting them into the preview itself.  That's a challenge, since these are functions, and as far as I know, can't be properly serialized.  But, maybe there's a way to do it?

## How to test


To see the issue, you can compile the code and create a vite sandbox.  When it tries to start up, you'll get an error about path only importing strings, and that's because the builder still expects a list of package names.
